.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Automatically stale issue and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close Stale Issues
+        uses: actions/stale@v9.0.0
+        with:
+          days-before-pr-close: 30
+          days-before-issue-stale: 30
+          days-before-issue-close: 10
+          stale-issue-message: 'After 30 days of inactivity, this issue has been marked as stale. This issue will be auto-closed in 10 days. Comment, assign or remove the stale label to prevent this.'
```

Comment:
`stale` label is automatically assigned after 30 days, so saying `assign or remove the stale label` sounds off

Reply:
Ah true, terminology is tough! Assign (in my mind) meant assign the PR to a person and not adding a label. All changes welcome!